app/models/crop.rb

```diff
@@ -41,6 +41,17 @@ def main_image_path
     if pictures.present?
       pictures.first.attachment.url
     else
+      # WARNING! MVC VIOLATION AHEAD!!!! =======================================
```

Comment:
Hey @simonv3 - this is that comment that you suggested we add.

I sat on it a little bit more and came to thinking that this would be a really good use case for a presenter object.

@warpling seems to hit a lot of use cases where he needs very complex data logic (that is too complex for a view). We've been shoving it into the model layer, but this might clutter up our classes after a while. Should have thought about this one sooner, but it might be worth a shot and shouldn't be too much work to migrate existing methods over.